src/views/Home/Home.js

```diff
@@ -1,18 +1,34 @@
 import React from 'react'
-
+import { useState } from 'react'
 import Preview from '../../components/Preview/Preview'
 import Editor from '../../components/Editor/Editor'
 
 import './Home.css'
 
 export default function Home() {
   // add useState calls here for title, subtitle, font, align, and text
+  const [title, setTitle] = useState('')
+  const [subtitle, setSubtitle] = useState('')
+  const [font, setFont] = useState('comforter')
+  const [align, setAlign] = useState('ri-align-left')
+  const [text, setText] = useState('')
 
   return (
     <main>
       {/* pass the state variables as props to the presentational components */}
-      <Preview />
-      <Editor />
+      <Preview title={title} subtitle={subtitle} font={font} align={align} text={text} />
+      <Editor
+        title={title}
+        onTitleChange={setTitle}
```

Comment:
If you use onTitleChange here when passing in the prop, you'll need to also use that terminology in the Editor. The way the Editor uses the props now, you'd want to pass in setTitle={setTitle}. For this and the other function props

Reply:
i had actually switched that but never pushed that commit. whoop whoop